source/opt/instruction.h

```diff
@@ -806,6 +810,14 @@ inline void Instruction::ForEachInOperand(
   });
 }
 
+inline bool Instruction::HasBranchWeights() const {
```

Comment:
Don't make this 'inline' - make it a normal method and put the implementation in the .cpp file.  (We can promote it to inline in due course if profiling suggests that it is a hotspot.)

Reply:
Done.